blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -555,6 +555,14 @@ public void testPutMultipartInputStream() throws Exception {
       testPut(payload, new ByteSourcePayload(byteSource), length, new PutOptions().multipart(true));
    }
 
+   /* Java on OS X does not support extended attributes, which the filesystem backend
+    * uses to implement user metadata */
+   private void checkUserMetadata(Blob blob, Map<String, String> userMetadata1, Map<String, String> userMetadata2) {
+      if ("filesystem".equals(blob.getMetadata().getProviderId()) && !TestUtils.isMacOSX()) {
```

Comment:
Can you make this method protected and override it in `FilesystemBlobIntegrationTest`?  We follow this pattern for other attributes like system metadata.